runtime-config-linux.md

```diff
@@ -144,6 +144,12 @@ Also known as cgroups, they are used to restrict resource usage for a container
 cgroups provide controls to restrict cpu, memory, IO, pids and network for the container.
 For more information, see the [kernel cgroups documentation](https://www.kernel.org/doc/Documentation/cgroups/cgroups.txt).
 
+There are a few facets to this:
+
+* Managing the existence of cgroups within the hierarchy (`cgroupsPath`).
+* Assigning processes to cgroups.
+* Configuring cgroups to apply resource limits (`resources`).
+
```

Comment:
I don't quite understand what's the meaning of "Managing the existence of cgroups within the hierarchy", does this includes:
- Assigning processes to cgroups (where to assign to)
- Configuring cgroups to apply resource limits (what kind of resources limits)

Because above two are almost all we can do to manage cgroup. I think the three facets are not on the same page, that's why I think is not clear.